atom/browser/api/atom_api_web_contents.cc

```diff
@@ -1884,6 +1884,16 @@ void WebContents::OnGetZoomLevel(content::RenderFrameHost* rfh,
   rfh->Send(reply_msg);
 }
 
+v8::Local<v8::Value> WebContents::GetPreloadPath(v8::Isolate* isolate) const {
+  base::FilePath::StringType preload;
+  if (auto* web_preferences = WebContentsPreferences::From(web_contents())) {
```

Comment:
_(trivial)_ declaration of `preload` could be moved inside this `if {` clause

Reply:
@ckerr done